src/sentry/issues/grouptype.py

```diff
@@ -383,6 +383,7 @@ class PerformanceDurationRegressionGroupType(GroupType):
     enable_auto_resolve = False
     enable_escalation_detection = False
     default_priority = PriorityLevel.LOW
+    notification_config = NotificationConfig(context=["Approx. Start Time"])
```

Comment:
might want to consider making a const for this value since it's repeated and easier for us to update in the future if product/design wants a change